diff --git a/fizz/tool/FizzServerCommand.cpp b/fizz/tool/FizzServerCommand.cpp
index f2e5990c4..c978c4d50 100644
--- a/fizz/tool/FizzServerCommand.cpp
+++ b/fizz/tool/FizzServerCommand.cpp
@@ -419,7 +419,7 @@ class FizzHTTPServer : public folly::AsyncTransportWrapper::ReadCallback {
         folly::io::Appender appender(response.get(), 10);
         std::string responseBody =
             transport_ ? respondHandshakeSuccess() : respondFallbackSuccess();
-        format(
+        folly::format(
             "HTTP/1.0 200 OK\r\n"
             "Content-Type: text/plain\r\n"
             "Content-Length: {}\r\n\r\n"
diff --git a/fizz/tool/FizzClientCommand.cpp b/fizz/tool/FizzClientCommand.cpp
index 97b76156f..f4b75b459 100644
--- a/fizz/tool/FizzClientCommand.cpp
+++ b/fizz/tool/FizzClientCommand.cpp
@@ -147,7 +147,7 @@ class Connection : public folly::AsyncSocket::ConnectCallback,
     if (!proxyTarget_.empty()) {
       auto connectCommand = IOBuf::create(0);
       folly::io::Appender appender(connectCommand.get(), 10);
-      format(
+      folly::format(
           "CONNECT {} HTTP/1.1\r\n"
           "Host: {}\r\n\r\n",
           proxyTarget_,
