diff --git a/bazel/gflags.bzl b/bazel/gflags.bzl
index 86a3d45..d35b0ee 100644
--- a/bazel/gflags.bzl
+++ b/bazel/gflags.bzl
@@ -1,3 +1,5 @@
+load("@rules_cc//cc:defs.bzl", "cc_library")
+
 # ------------------------------------------------------------------------------
 # Add native rules to configure source files
 def gflags_sources(namespace=["google", "gflags"]):
@@ -95,7 +97,7 @@ def gflags_library(hdrs=[], srcs=[], threads=1):
     else:
         name += "_nothreads"
         copts += ["-DNO_THREADS"]
-    native.cc_library(
+    cc_library(
         name       = name,
         hdrs       = hdrs,
         srcs       = srcs,
